webviews/welcome/index.css

```diff
@@ -0,0 +1,15 @@
+h1 {
+    font-size: 22px
+}
+
+h2 {
+    font-size: 18px;
+}
+
+h2 .css-style {
+    color: #ffc107;
```

Comment:
it seems like vs code webview has a means to alter colors depending on the current theme - does this display well for different themes?

Reply:
@dkelosky This's just an example. We won't change default color without strong necessity.

Reply:
@stepanzharychevbroadcom Opening a Draft PR is better for this so the community won't start reviewing this if this is not ready for review.

Reply:
i'm still confused if this is an example or if this PR will go in as-is.  it seems like the colors should be set based on the current [theme](https://code.visualstudio.com/api/extension-guides/webview#theming-webview-content)